free-2.md

```diff
@@ -58,12 +114,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think to make this an effective language, good research should go into the 
+science behind having a period, the common terminology, and how to take the 
+more specific medical language and translate it to something more accessible
+in everday vernacular. Also, deciding what information is necessary and what
+information is auxillary in tracking a period would take both research and
+consideration. I think there could be some technical challenges if a lot of 
+work went into taking this data and making predictions, but I think just
+tracking it would be more technically simple.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+If it's literally just tracking the days you are/aren't on your period, 
+then this could be an incredibly easy project (doable in less than an hour).
+However, adding more detail to the type of data a user can manage and what 
+they could do with that detail opens up a lot of possibilities and room
+for this to grow.
+
 
```

Comment:
I agree and I like the flexibility of how you could stretch it or shrink it defending on time constraints.